.circleci/config.yml

```diff
@@ -111,15 +111,15 @@ jobs:
 
             inv project.ci-build
       - when:
-        # If this test is not being run on master or develop, then we should
+        # If this test is not being run on main or develop, then we should
```

Comment:
I wasn't sure about this one though because it was my personal choice to use a `main` branch, while I think a lot of projects are still using `master`. But I'll raise it over there.

Reply:
Makes sense. Since GitHub's default is now `main`, it seems like that's a better default for jade-truffle too? But we can move the conversation to there. Either way, nothing about it needs to change in this pull request.

Reply:
Agree, it is a great suggestion. I created an issue over there: https://github.com/caktus/jade-truffle/issues/16